lib/wasm/WebAssemblyJavascriptGenerator.js

```diff
@@ -18,7 +18,25 @@ const WebAssemblyImportDependency = require("../dependencies/WebAssemblyImportDe
 /** @typedef {import("../NormalModule")} NormalModule */
 /** @typedef {import("../RuntimeTemplate")} RuntimeTemplate */
 
+const TYPES = new Set(["webassembly"]);
+
 class WebAssemblyJavascriptGenerator extends Generator {
+	/**
+	 * @returns {Set<string>} available types (do not mutate)
+	 */
+	getTypes() {
+		return TYPES;
+	}
+
+	/**
+	 * @param {NormalModule} module the module
+	 * @param {string=} type source type
+	 * @returns {number} estimate size of the module
+	 */
+	getSize(module, type) {
+		return 100 + module.dependencies.length * 5;
```

Comment:
What's the reasoning behind this formula?

Reply:
That's about the code size of the generated code for WebAssembly modules.